docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+[buildConfigSource](#build_config_source) | object | The resolved reference to the external build config source.
+
+### Build Config Source
+Parameter | Type | Description
+-----------|------|------------
+ref | string | <li>git reference (i.e. commit sha - <code>"sha1:xxxx"</code>)</li><li>image digest (e.g. <code>"sha256:91fb5e20325059e8feb48c966a83d616596054c5edf811b5bc673683e6ecabb6"</code>)</li>
+repository | string | git repository uri or image repository uri
+path | string | path in the git repo or the resource name in the image bundle.
+
+## Internal Parameters
+
+These are fields provided by the builder for the build process. 
+
+Parameter | Type | Description
+-----------|------|------------
+[FeatureFlags](https://tekton.dev/docs/pipelines/additional-configs/#customizing-the-pipelines-controller-behavior)| object | Tekton configuration feature flags used for the build. These are configured by the service providers and are useful for reproducibility.
+
+## Resolved Dependencies
+
+The resolvedDependencies contains artifacts known to be input to the build. And the name will be used to indicate the type of the artifact. The name of the resolved dependencies is used to distinguish between the different input types to the build as shown below:
+
+Name | Description
+-----|-------------
+**pipeline**|Resolved reference to the remote pipeline.
+**task**|Resolved reference to remote tasks. This name is used for a build performed via a direct `TaskRun`.
+**pipelineTask**|Resolved reference to remote `PipelineTasks`.
+N/A|Resolved reference to `step` and `sidecar container images`.<li>Even if the images are referenced by a tag in the spec it will be resolved and its digest will be included.</li><li>This resolved dependency is not named because it does not add anything of value that is needed right now from step/sidecar images.</li>
+**input/results**|Input Artifacts used for the build, e.g. `Source code`. These are type-hinted results emitted by the task.
+
+
+```json
+{
+    "resolvedDependencies": [
+    {
+        "name": "config/pipeline",
+        "uri": "git+https://github.com/octocat/hello-world.git",
+        "digest": {"sha1": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"},
+    }, {
+        "name": "config/pipelineTask",
+        "uri": "git+https://github.com/octocat/hello-world.git",
+        "digest": {"sha1": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"},
+    }, {
+         "uri": "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init",
+         "digest": {
+             "sha256": "28ff94e63e4058afc3f15b4c11c08cf3b54fa91faa646a4bbac90380cd7158df"},
+    }, {
+        "name": "input/result",
+        "uri": "git+https://github.com/octocat/hello-world.git",
+        "digest": {"sha1": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"},
+    }
+    ]
+}
+```
+
+
+
+# Run Details
+
+
+## Builder
+
+
+### BuilderId
+
+URI provided by the builder using Tekton Chains to generate the provenance.
+
+
+### Version
+
+TBD
```

Comment:
Another subfield under builder is `builderDependencies`. Maybe mention both builderDependencies and version are empty?